.changeset/rare-pumpkins-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/app': patch
+---
+
+Fix issue 23504 - Set initial state of the tune in ImageTool renderSettings()
```

Comment:
Let's be a little less specific here and let's not mention the issue number, it will already be tractable through the linked PR in the changeset

Reply:
Sure. I adjusted the changeset accordingly